systems/subscriber/sim-pool/pkg/server/event.go

```diff
@@ -0,0 +1,48 @@
+package server
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+	epb "github.com/ukama/ukama/systems/common/pb/gen/events"
+	pb "github.com/ukama/ukama/systems/subscriber/sim-pool/pb/gen"
+	"github.com/ukama/ukama/systems/subscriber/sim-pool/pkg/db"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type SimPoolEventServer struct {
+	simPoolRepo db.SimRepo
+	epb.UnimplementedEventNotificationServiceServer
+}
+
+func NewSimPoolEventServer(simPoolRepo db.SimRepo) *SimPoolEventServer {
+	return &SimPoolEventServer{
+		simPoolRepo: simPoolRepo,
+	}
+}
+func (l *SimPoolEventServer) EventNotification(ctx context.Context, e *epb.Event) (*epb.EventResponse, error) {
+	log.Infof("Received a message with Routing key %s and Message %+v", e.RoutingKey, e.Msg)
+	switch e.RoutingKey {
+	case "event.cloud.simManager.sim.allocation":
+	default:
+		log.Errorf("handler not registered for %s",e.RoutingKey)
+	}
+
+	return &epb.EventResponse{}, nil
+}
+
+func unmarshalAllocateSim(msg *anypb.Any) (*pb.UploadRequest, error) {
```

Comment:
check the warning

Reply:
![image](https://user-images.githubusercontent.com/24965604/215590172-5971cf18-4cfa-424f-a255-924baf38f057.png)

these type of warnings by github has to be resolved. let me know if you still have difficulty in seeing them